### PR TITLE
Flake8

### PR DESCRIPTION
**Pull request by Sharad24**
- Flake8 pre-commit checks
 - setup.cfg 
 - isort pre-commit checks

### CODE REVIEW

File: tests/test_classical/test_agents.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+from genrl import SARSA, QLearning
 
 
 class TestAgents:
     def test_qlearning(self):
         env = gym.make("FrozenLake-v0")
         agent = QLearning(env)
 
-        assert np.any(agent.Q) == False
+        assert np.any(agent.Q) is not True
```
Comment (Sharad24) [inline tests/test_classical/test_agents.py:13]: Am I missing out on any edge cases here? Can someone take a look?

I've just changed `== False` to `is not True`

The reason I'm asking is because with just keeping `is False`, the test fails.

Comment (Zeus3101) [reply tests/test_classical/test_agents.py:13]: It could be `None` too, that's not likely though. It's simpler to write `assert not np.any(agent.Q)`

File: genrl/__init__.py [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-    BanditPolicy,
-    EpsGreedyPolicy,
-    UCBPolicy,
-    SoftmaxActionSelectionPolicy,
-    BayesianUCBPolicy,
-    ThompsonSamplingPolicy,
-    SARSA,
-    QLearning,
-)
+from genrl.classical import SARSA  # noqa; noqa; noqa
```
Comment (Zeus3101) [inline genrl/__init__.py:1]: Why are there 3 separate `noqa`s? And is it actually necessary to separate imports like this?

Comment (Sharad24) [reply genrl/__init__.py:1]: Lol, I have no clue where this came from. I just added 1

**Comment by Sharad24** [comment]
Going to merge this, if there are no more comments.

**Comment by ajaysub110** [comment]
Cool, looks fine
